R/print.R

```diff
@@ -9,28 +9,21 @@ format.formula_blueprint <- function(x, ...) "Formula"
 
 #' @export
 print.hardhat_blueprint <- function(x, ...) {
-  cat_line("{format(x)} blueprint:")
-  cat_line("\n")
-  cat_line("# Predictors: {n_blueprint_predictors(x)}")
-  cat_line("  # Outcomes: {n_blueprint_outcomes(x)}")
-  cat_line("   Intercept: {x$intercept}")
-  cat_line("Novel Levels: {x$allow_novel_levels}")
-  cat_line(" Composition: {x$composition}")
-  invisible(x)
-}
+  cli::cli_text("{format(x)} blueprint:")
 
-#' @export
-print.formula_blueprint <- function(x, ...) {
-  NextMethod()
-  cat_line("  Indicators: {x$indicators}")
+  cli::cli_par()
+  cli::cli_text("# Predictors: {n_blueprint_predictors(x)}")
+  cli::cli_text("# Outcomes: {n_blueprint_outcomes(x)}")
+  cli::cli_text("Intercept: {x$intercept}")
+  cli::cli_text("Novel Levels: {x$allow_novel_levels}")
+  cli::cli_text("Composition: {x$composition}")
+  if (inherits(x, "formula_blueprint")) {
+    cli::cli_text("Indicators: {x$indicators}")
+  }
```

Comment:
Moving this inside the parent method so that it can live within the same cli paragraph.